### PR TITLE
Integrate IO Chart

### PR DESCRIPTION
**Pull request by danieldortega77**
Integrate the components from the analysis page, Java servlet, and JavaScript. Error handling for attribute selection and languages needs to be added in a future PR.

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,6 +12,21 @@
...
+async function submitInput() {
+  const textElement = document.getElementById('textForAnalysis');
+  if (!textElement) {
+    return;
+  }
+  const langElement = document.getElementById('languageForAnalysis');
+  if (!langElement) {
+    return;
+  }
+  const response = await callPerspective(textElement.value, langElement.value);
```
Comment (chrismohrii) [inline capstone/src/main/webapp/script.js]: Could we start this line at `await...`? Seems like we don't use the `response` constant

**Review by chrismohrii — APPROVED** [review]
LGTM

File: capstone/src/main/webapp/script.js (hunk 1 above)
Comment (danieldortega77) [reply capstone/src/main/webapp/script.js]: Yes, that's right - I'll remove the `response`.

**Comment by danieldortega77** [comment]
#13 contains all of these features and is further in the review process

**Pull request closed without merging**